addons/project/static/tests/tours/project_task_history.js

```diff
@@ -112,6 +112,40 @@ registry.category("web_tour.tours").add("project_task_history_tour", {
                 throw new Error(`Expect description to be ${expected}, got ${p}`);
             }
         }
+    }, {
+        content: "Go back to projects view.",
+        trigger: 'a[data-menu-xmlid="project.menu_projects"]',
+    }, {
+        content: "Open Test History Project Without Tasks",
+        trigger: "div span.o_text_overflow[title='Without tasks project']",
+        extra_trigger: ".o_kanban_view",
+    }, {
+        content: "Switch to list view",
+        extra_trigger: "div .o_project_task_kanban_view",
+        trigger: ".o_switch_view.o_list",
+    }, {
+        content: "Create a new task.",
+        trigger: '.o_list_button_add',
+    }, {
+        trigger: 'textarea[id="name_0"]',
+        content: 'Set task name',
+        run: 'text New task',
+    },
+        ...changeDescriptionContentAndSave("0"),
+        ...changeDescriptionContentAndSave("1"),
+        ...changeDescriptionContentAndSave("2"),
+        ...changeDescriptionContentAndSave("3"),
+    {
+        content: "Open History Dialog",
+        trigger: ".o_cp_action_menus i.fa-cog",
+        extra_trigger: ".o_form_view",
+    }, {
+        content: "Open History Dialog",
+        trigger: ".o_cp_action_menus .o_menu_item i.fa-history",
+        extra_trigger: ".o_cp_action_menus .dropdown-menu",
```

Comment:
```suggestion
        trigger: ".o_menu_item i.fa-history",
        extra_trigger: ".dropdown-menu",
```
i copy paste from above, not test yet hihi

Reply:
thanks 🙂